src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests.csproj

```diff
@@ -44,5 +50,8 @@
       <Name>RemoteExecutorConsoleApp</Name>
     </ProjectReference>
   </ItemGroup>
+  <Target Condition="'$(TargetsWindows)' == 'true'" Name="BuildNetfxRemoteInvoker" AfterTargets="BeforeBuild">
+    <MSBuild Projects="RemoteInvokeNetfx\RemoteInvokeNetfx.csproj" Properties="Configuration=Debug; Platform=$(Platform); OutputPath=$(OutputPath); IntermediateOutputPath=$(IntermediateOutputPath)" />
```

Comment:
@joperezr it is more complicated than what you think here. RemoteInvoke in UAP is activating the APPX package and not creating or using Process. 

S.D.Process is a special case as we need to test the Process class. hacking RemoteInvoke will complicate it while we need to have a solution for this test only. the simpler way is just to use desktop app to test the Process class with it. 

Let's talk to Wes offline